debian/changelog

```diff
@@ -0,0 +1,5 @@
+golang-logrus (0.6.0) UNRELEASED; urgency=medium
+
+	* Initial release.
```

Comment:
Also, we traditionally use two spaces rather then a tab, but that's not written anywhere